app/assets/stylesheets/custom.scss

```
@@ -39,3 +45,72 @@ textarea {
   color: $white;
   text-decoration: none;
 }
+
+/* sidebar */
+aside {
+  section.user_info {
+    margin-top: 20px;
+  }
+
+  section {
+    padding: 10px 0;
+    margin-top: 20px;
+
+    &:first-child {
+      border: 0;
+      padding-top: 0;
+    }
+
+    span {
+      display: block;
+      margin-bottom: 3px;
+      line-height: 1;
+    }
+    h1 {
+      font-size: 1.4em;
+      text-align: left;
+      letter-spacing: -1px;
+      margin-bottom: 3px;
+      margin-top: 0px;
+    }
+  }
+}
+
+.gravatar {
+  float: left;
+  margin-right: 10px;
+}
+.gravatar_edit {
+  margin-top: 15px;
+}
+
+/* forms */
+input, textarea, select, .uneditable-input {
+  border: 1px solid $grey;
+  width: 100%;
+  margin-bottom: 15px;
+  @include box_sizing;
+}
+input {
```

giữa các selector với nhau thì phải có 1 line trống nhé bạn
```suggestion
}

input {
```